packages/remix-dev/compiler/plugins/cssModulesPlugin.ts

```diff
@@ -17,14 +19,20 @@ interface PluginData {
   compiledCss: string;
 }
 
-export const cssModulesPlugin = (options: {
+export const cssModulesPlugin = ({
+  config,
+  mode,
+  outputCss,
+}: {
+  config: RemixConfig;
   mode: CompileOptions["mode"];
-  rootDirectory: string;
   outputCss: boolean;
 }): Plugin => {
   return {
     name: pluginName,
     setup: async (build: PluginBuild) => {
+      let postcssPlugins = await loadPostcssPlugins({ config });
+
       build.onResolve(
         { filter: cssModulesFilter, namespace: "file" },
```

Comment:
Same note here about `.scss`, I wonder if supporting sass modules would be this easy...